contrib/world_builder/source/world_builder/world.cc

```diff
@@ -304,6 +304,7 @@ namespace WorldBuilder
     // We receive the cartesian points from the user.
     const Point<3> point(point_,cartesian);
 
+    (void) this->limit_debug_consistency_checks;
     WBAssert(!this->limit_debug_consistency_checks || this->parameters.coordinate_system->natural_coordinate_system() == cartesian
```

Comment:
Does it really warn about this variable being unused? It's a member variable, how does the compiler know that the function isn't used anywhere else?

Reply:
In debug mode is isn't used anywhere. I never got this warning, but it does appear on some mac's apparently.

Reply:
Fascinating.